### DISCUSSION TITLE
脱敏失败 && 下载v1.10.0版本的zip包，启动后是1.9的版本

### DISCUSSION THREAD

**Discussion by Conanke888** [hhyo/Archery#2451] [General]
俩个问题
问题1：脱敏失败
说明：
在系统配置里goinception 4000连通是正常的
![image](https://github.com/hhyo/Archery/assets/62231406/245500fc-97df-46a1-9932-5bdca7b09919)

脱敏字段配置：
![image](https://github.com/hhyo/Archery/assets/62231406/4b6e45c8-ce0c-43db-9d4a-b98b4c69ebd1)
脱敏规则：
![image](https://github.com/hhyo/Archery/assets/62231406/9e5ac900-5494-49f5-9b13-8296890ef9d8)

系统配置里的脱敏两个选项DATA_MASKING QUERY_CHECK 已经打开

这么配置之后在页面查询的时候，字段还是没有脱敏 这个什么原因？？

问题2：下载1.10版本失败
在https://github.com/hhyo/Archery/releases/tag/v1.10.0 目录页面，assets下下载的包 ，
启动服务之后发现是1.9的版本 这个1.10的版本怎么下

**Comment by LeoQuote** [comment]
https://github.com/hhyo/archery/wiki/upgrade#docker-%E5%AE%89%E8%A3%85


https://github.com/hhyo/archery/wiki/faq#%E8%84%B1%E6%95%8F%E6%9F%A5%E8%AF%A2%E8%A7%84%E5%88%99%E6%9C%AA%E7%94%9F%E6%95%88
